app/controllers/answers_controller.rb

```diff
@@ -37,6 +37,11 @@ def destroy
     end
   end
 
+  def accept
+    Answer.accept(params[:answer_id])
```

Comment:
The `accept` can not be an instance method, because it is calling the `update_user_reputation` class method. The later is made a class method because the method that calls it from the `Vote` model has been made a class method. ¯\_(ツ)_/¯